encode_datasets.py

```diff
@@ -1535,13 +1537,12 @@ def record_process_extractive (file,dataset, kind):
                     question = question + "."  
                 if kind is not "test":     
                     answer_string=json_line['qas'][index]['answers'][0]['text'].strip().replace("\n", "").replace("\t", "").replace("   ", " ").replace("  ", " ")
+                else:
+                    answer_string="-"
                 id=json_line['qas'][index]['idx']
 
                 fmeta.write(f"{id}\n")
```

Comment:
Similarly here, can you output all the valid answers to the 2nd column of meta file, for `record_extractive` dataset?